sources/Inputs/include/Configuration.h

```diff
@@ -169,7 +176,35 @@ class Configuration {
     return assemblingFilePath_;
   }
 
+  /**
+   * @brief Chosen outputs
+   *
+   * struct that contains all chosen outputs
+   */
+  struct ChosenOutputs {
+    bool isSteadystateChosen;  ///< STEADYSTATE output is chosen
+    bool isConstraintsChosen;  ///< CONSTRAINTS output is chosen
+    bool isLostEqChosen;       ///< LOSTEQ output is chosen
+    bool isTimelineChosen;     ///< TIMELINE output is chosen
+  };
+
+  /**
+   * @brief Retrieves the chosen outputs
+   * @returns the chosen outputs
+   */
+  const ChosenOutputs& getChosenOutputs() const {
```

Comment:
I was thinking of a struct ChosenOutputs {STEADYSTATE, CONSTRAINTS ...} and bool isChosenOutputs(ChosenOutputs type) const;